### PR TITLE
15 - add interactions and styling to application outline tabs

### PR DESCRIPTION
**Pull request by caravinci**
bonus:
- fix favicon path being relative to pages
- fix Navbar's shadow being covered by content elements

### CODE REVIEW

File: components/Head.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Head = () => {
         href={'https://fonts.googleapis.com/css?family=Lato:300,400,600&display=swap'}
         rel="stylesheet"
       />
-      <link rel="shortcut icon" href={`favicon.ico`} />
+      <link rel="shortcut icon" href="/favicon.ico" />
```
Comment (caravinci) [inline components/Head.tsx:11]: This should fix the favicon issue.

File: components/pages/Applications/ApplicationForm/Header/Progress.tsx [hunk 2]
```diff
@@ -1,7 +1,5 @@
-import { ReactElement } from "react";
+import { ReactElement } from 'react';
```
Comment (caravinci) [inline components/pages/Applications/ApplicationForm/Header/Progress.tsx:1]: Linting only...

**Review by caravinci — COMMENTED** [review]
Some comments to try to give context, but I realise there are several changes.
soooo... TL;DR: customisation of the tabs, and a whole lot of linting.

File: components/NavBar.tsx [hunk 3]
```diff
@@ -47,6 +47,7 @@ const NavBar = () => {
           display: flex;
           justify-content: space-between;
           box-shadow: 0 1px 6px 0 rgba(0, 0, 0, 0.1), 0 1px 5px 0 rgba(0, 0, 0, 0.08);
+          z-index: 1;
```
Comment (caravinci) [inline components/NavBar.tsx:50]: and this fixes the navbar shadow being covered

**Review by samrichca — APPROVED** [review]

**Pull request merged**